package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0.0",
   "description": "Universal Redux Template",
   "scripts": {
-    "start": "NODE_PATH=./app nodemon app/server & node webpack.server.js & gulp css:watch",
-    "test": "NODE_ENV=test NODE_PATH=./app mocha --compilers js:babel-register -r app/spec/support/setup.mocha.js --recursive 'app/spec/**/*.test.js' -w",
+    "start": "cross-env NODE_PATH=./app concurrent --kill-others \"nodemon app/server\" \"node webpack.server.js\" \"npm run css:watch\"",
+    "test": "cross-env NODE_ENV=test cross-env NODE_PATH=./app mocha --compilers js:babel-register -r app/spec/support/setup.mocha.js --recursive 'app/spec/**/*.test.js' -w",
     "webpack": "webpack",
-    "build": "rm -rf dist/; npm run webpack; gulp css:build",
+    "build": "rimraf dist/ & npm run webpack & gulp css:build",
```

Comment:
I changed it to `&&` could you check to see if that works for you? `;` will not work in windows